dev-python/pydispatcher/pydispatcher-2.0.1.recipe

```diff
@@ -1,25 +1,63 @@
-DESCRIPTION="
-Multi-producer-multi-consumer signal dispatching mechanism in python.
-"
+SUMMARY="Multi-producer-multi-consumer signal dispatching mechanism in Python"
+DESCRIPTION="PyDispatcher provides the Python programmer with a \
+multiple-producer-multiple-consumer, signal-registration and routing infrastructure for \
+use in multiple contexts. The mechanism of PyDispatcher started life as a highly rated \
+recipe in the Python Cookbook. The SourceForge project aims to include various \
+enhancements to the recipe developed during use in various applications."
 HOMEPAGE="http://pydispatcher.sourceforge.net/"
+COPYRIGHT="2001-2006, Patrick K. O'Brien and Contributors"
+LICENSE="BSD (3-clause)"
+REVISION="2"
 SOURCE_URI="http://downloads.sourceforge.net/project/pydispatcher/pydispatcher/2.0.1/PyDispatcher-2.0.1.tar.gz"
-REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND="dev-python/setuptools >= 0.6"
-CHECKSUM_MD5="8b1695ed6f5bdbda8fbcde82b62d505c"
+CHECKSUM_SHA256="a126561446cd1c9fdee0d43a32cae4413831dc017a64baf2f2fecd1597203898"
+SOURCE_FILENAME="PyDispatcher-2.0.1.tar.gz"
```

Comment:
```diff
 CHECKSUM_SHA256="a126561446cd1c9fdee0d43a32cae4413831dc017a64baf2f2fecd1597203898"
-SOURCE_FILENAME="PyDispatcher-2.0.1.tar.gz"
-SOURCE_DIR="PyDispatcher-2.0.1"
+SOURCE_DIR="PyDispatcher-$portVersion"
+
 ARCHITECTURES="any"
 
 PROVIDES="
```
* `SOURCE_FILENAME` is not required because its default value is ok.
* We like to use `$portVersion` in `SOURCE_DIR`.
* We like to have a blank line before `ARCHITECTURES`.

Refs: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)